xbmc/platform/Platform.cpp

```diff
@@ -19,6 +19,9 @@
  */
 
 #include "Platform.h"
+#if HAS_LIRC
+#include "platform/linux/input/LIRC.h"
```

Comment:
Can this be moved to PlatformLinux so we don't have a linux include in the generic implementation?

Reply:
moved to all platforms which depends on it. thanks